app/uk/gov/hmrc/agentservicesaccount/models/ManageAccessPermissionsConfig.scala

```diff
@@ -33,9 +33,9 @@ object ManageAccessPermissionsConfig {
   def apply(optinStatus: OptinStatus, hasAnyGroups: Boolean)(implicit appConfig: AppConfig): ManageAccessPermissionsConfig = optinStatus match {
     case OptedInReady => ManageAccessPermissionsConfig(
       status = "manage.account.manage-access-permissions.status-opted-in",
-      statusClass = "govuk-tag",
-      insetText = Some("manage.account.manage-access-permissions.inset-text.Opted-In_READY"),
-      explainAccessGroups = !hasAnyGroups,
+      statusClass = "govuk-body govuk-!-margin-left-2",
+      insetText = None,
+      explainAccessGroups = true,
```

Comment:
if this is always true why keep the boolean?

Reply:
It didn't use to be true for all pages but one by one the new design all started to include that explanation.
Since it's there I'd keep it as designers are liable to change their mind

Reply:
personally I'd remove it since it's a bit messy but I guess we can tidy it up later as this won't be the last update to the page - I just don't want to forget aha